content/terraform/v1.14.x (alpha)/docs/language/block/tfquery/list.mdx

```diff
@@ -0,0 +1,321 @@
+---
+page_title: list block reference for `.tfquery.hcl` files
+description: |-
+ Add the `list` block to your `.tfquery.hcl` files to declare Terraform providers and configure them when querying existing infrastructure for resources to import. 
+---
+
+# `list` block reference
+
+The `list` block defines a query that reads existing infrastructure and returns unmanaged resources that you can import to your Terraform workspace. You can only add `list` blocks to configuration files with `.tfquery.hcl` extensions. This topic provides reference information about configuring `list` blocks in `.tfquery.hcl` files. Refer to [Import existing resources in bulk](/terraform/language/import/bulk) for information about querying and importing resources. 
```

Comment:
Just the same note, double check the language around "unmanaged resources" since I think _technically_ it can return managed resources.